scripts/cat_relations/relationship.py

```diff
@@ -443,272 +454,295 @@ def get_interaction_type(self, positive: bool) -> str:
+        elif group == "mid_neg":
+            return ValueLevel.ENVIES
+        elif group == "low_neg":
+            return ValueLevel.BEGRUDGES
+        elif group == "neutral":
+            return ValueLevel.ACKNOWLEDGES
+        elif group == "low_pos":
+            return ValueLevel.PRAISES
+        elif group == "mid_pos":
+            return ValueLevel.RESPECTS
+        elif group == "extreme_pos":
+            return ValueLevel.ADMIRES
+        else:
+            return None
 
-    @jealousy.setter
-    def jealousy(self, value):
+    @property
+    def comfort(self) -> int:
+        return self._comfort
+
+    @comfort.setter
+    def comfort(self, value):
         if value > 100:
             value = 100
-        if value < 0:
-            value = 0
-        self._jealousy = value
+        if value < -100:
+            value = -100
+        self._comfort = value
 
     @property
-    def trust(self):
+    def comfort_level(self) -> Optional[ValueLevel]:
+        group = self._get_level_group(self.comfort)
+
+        if group == "extreme_neg":
+            return ValueLevel.RUNS_FROM
+        elif group == "mid_neg":
+            return ValueLevel.FEARS
+        elif group == "low_neg":
+            return ValueLevel.AVOIDS
+        elif group == "neutral":
+            return ValueLevel.CONSIDERS
+        elif group == "low_pos":
+            return ValueLevel.RELATES_TO
+        elif group == "mid_pos":
+            return ValueLevel.UNDERSTANDS
+        elif group == "extreme_pos":
+            return ValueLevel.KNOWS_DEEPLY
+        else:
+            return None
+
+    @property
+    def trust(self) -> int:
         return self._trust
 
     @trust.setter
     def trust(self, value):
         if value > 100:
             value = 100
-        if value < 0:
-            value = 0
+        if value < -100:
+            value = -100
         self._trust = value
+
+    @property
+    def trust_level(self) -> Optional[ValueLevel]:
+        group = self._get_level_group(self.trust)
+
+        if group == "extreme_neg":
+            return ValueLevel.LOATHES
+        elif group == "mid_neg":
+            return ValueLevel.DISTRUSTS
+        elif group == "low_neg":
+            return ValueLevel.DOUBTS
+        elif group == "neutral":
+            return ValueLevel.OBSERVES
+        elif group == "low_pos":
+            return ValueLevel.LISTENS_TO
+        elif group == "mid_pos":
+            return ValueLevel.TRUSTS
+        elif group == "extreme_pos":
+            return ValueLevel.CONFIDES_IN
+        else:
+            return None
+
+    @staticmethod
+    def _get_level_group(value) -> str:
+        """
+        Returns the level group for the given value.
+        """
+        found_group = None
+        for group, interval in game.config["relationship"]["value_intervals"].items():
+            if value <= interval:
+                found_group = group
+                break
+        return found_group
```

Comment:
```suggestion
                return group
```
and get rid of `found_group`